dockers/base-cuda/Dockerfile

```diff
@@ -75,6 +75,8 @@ ENV \
 COPY ./requirements.txt requirements.txt
 COPY ./requirements/ ./requirements/
 
+ENV PYTHONPATH=/usr/lib/python${PYTHON_VERSION}/site-packages
```

Comment:
Curious, why is this preferred to `--user` and how is it related to the problem?

Reply:
the collision is that for some reason `--user` sometomes installs to `site-packages` others `dist-packages` but the was missing in paths `site-packages`